### DISCUSSION TITLE
Splitting a manuscript HTML into multiple web pages (similar to the book layout)

### DISCUSSION THREAD

**Discussion by apcamargo** [quarto-dev/quarto-cli#11718] [Q&A]
### Description

I’m working on a lengthy manuscript in Quarto that includes supplemental content. To keep things organized and avoid having one massive page, I’d like to separate the supplemental content from the main manuscript. Is there a way to organize the manuscript HTML into multiple separate pages?

I am aware that a book project could accomplish this, but I would prefer to stick with a manuscript project because of its helpful features for scientific papers.

**Comment by mcanouil** [comment]
That's not the purpose of Manuscript.

You could use the include shortcode to split the source of your manuscript.
